test/suites/homeshick_dir.bats

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bats
+
+load ../helper
+
+@test 'bash override homeshick_dir' {
+	castle 'dotfiles'
+	local result=$( HOMESHICK_DIR=$_TMPDIR/nowhere $HOMESHICK_FN 2>&1 >/dev/null )
+	echo "result=$result"
```

Comment:
Ah! OK, that's actually really nice. I'll remember that next time I write some tests :-)